### PR TITLE
Hello Microverse Set Up Project

### PR DESCRIPTION
**Pull request by haliljon**
In this pull request, I was able to:
  -   set up the linters
  -   create index file and add `h1` tag with `Hello Microverse` as content.
  -   create css file to style the content of the index file.
  -   create test.md file and add it to `.gitignore` file.
  -  etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Getting Started
+
+**This project is made solely for educaitonal purposes.**
```
Comment (ha-manel) [inline README.md]: - [x] This section is supposed to tell the user how to set up your project locally and how to open it. Kindly follow this [template](https://github.com/microverseinc/readme-template) and fill out the subtitles (prerequisites, setup, install...) 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @haliljon👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Github flow is followed correctly ✔️

## Required Changes:
- [ ] Kindly update the pull request's title, it should have a descriptive title telling you what changes are inside.
- [ ] Kindly update the pull request's description, it should have a summary describing the introduced changes in more detail.
- [ ] From now on, choose the branch's name more carefully, it should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section').

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (haliljon) [reply README.md]: Ok, I'm on it

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada:  :heavy_check_mark: 

Hi @haliljon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**kindly check an optional comment I left under the review and work on them**_ :pray: 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark:



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [OPTIONAL] I would like to recommend that you should always try to give your pull request a descriptive titile and summary. A title like `Hello Microverse Set Up Project` would have been better to `Amend and add files`. In the description section, you're expected to state what you've worked on in the project. Take the format below as an example

```
  In this pull request, I was able to:
  -   set up the linters
  -   create index file and add `h1` tag with `Hello Microverse` as content.
  -   create css file to style the content of the index file.
  -   create test.md file and add it to `.gitignore` file.
  -  etc.
```
- Kindly modify both the title and the summary section of your project for future reference.

**Comment by haliljon** [comment]
> # _Status: Approved_ 🎉 ✔️
> Hi @haliljon,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> _**kindly check an optional comment I left under the review and work on them**_ 🙏
> 
> ## Highlights
> * No linter errors. ✔️
> * Correct GitHub Flow. ✔️
> * Good README file. ✔️
> 
>  [  ](https://user-images.githubusercontent.com/55918904/191301390-43d62dfa-bfdc-49c5-943b-4d72f4ea5bb8.gif) [ ](https://user-images.githubusercontent.com/55918904/191301390-43d62dfa-bfdc-49c5-943b-4d72f4ea5bb8.gif)
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
> 
> * [OPTIONAL] I would like to recommend that you should always try to give your pull request a descriptive titile and summary. A title like `Hello Microverse Set Up Project` would have been better to `Amend and add files`. In the description section, you're expected to state what you've worked on in the project. Take the format below as an example
> 
> ```
>   In this pull request, I was able to:
>   -   set up the linters
>   -   create index file and add `h1` tag with `Hello Microverse` as content.
>   -   create css file to style the content of the index file.
>   -   create test.md file and add it to `.gitignore` file.
>   -  etc.
> ```
> 
> * Kindly modify both the title and the summary section of your project for future reference.

Thank you for correcting me. 🙏

**Pull request merged**